### PR TITLE
chengscott Term Proposal

### PR DESCRIPTION
**Pull request by chengscott**

### CODE REVIEW

File: term_proposal/chengscott/proposal.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+The PageRank equation (matrix version) is:
+$$
+r = A \cdot r
+$$
+
+The calculation requires several passes of iterations to approximate PageRank values.
+
+## Project
+
+In this project, I am going to implement PageRank algorithm with different kinds of BLAS. I expect to get familiar with several BLAS libraries, and compare performance among mine implementations. The code will all be written in modern C++, and be tested with small and large Dataset.
```
Comment (yungyuc) [inline term_proposal/chengscott/proposal.md]: This is a good description of the implementation.  How do you expect your users to use this project?

File: term_proposal/chengscott/proposal.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# nsd PageRank
+
+## Background
+
+PageRank was an algorithm used by Google to rank the importance of resulting web pages from their search engine. PageRank simulates Web surfers, starting at a random page and randomly browsing around, would tend to converge and aggregated, and this process is allowed to iterate many times.
+
+The algorithm accepts a directed graph as the input, where vertices and edges represents web pages and links between, respectively; It outputs PageRank scores for each vertices, which represents the likelihood of a page being visited by random web suffering. The problem can be reformulated in matrix calculations:
```
Comment (yungyuc) [inline term_proposal/chengscott/proposal.md]: Did you mean to say suffering or surfing?

**Review by yungyuc — CHANGES_REQUESTED** [review]

File: term_proposal/chengscott/proposal.md (hunk 2 above)
Comment (chengscott) [reply term_proposal/chengscott/proposal.md]: Thank you. surfing, it's a typo.

File: term_proposal/chengscott/proposal.md (hunk 1 above)
Comment (chengscott) [reply term_proposal/chengscott/proposal.md]: a possible use case is added, please review

**Pull request merged**